lib/commands/touch.js

```diff
@@ -0,0 +1,124 @@
+import log from '../logger';
+import _ from 'lodash';
+import androidHelpers from '../android-helpers';
+import { asyncmap } from 'asyncbox';
+
+let commands = {}, helpers = {}, extensions = {};
+
+helpers.parseTouch = async function (gestures, multi) {
+  // because multi-touch releases at the end by default
+  if (multi && _.last(gestures).action === 'release') {
+    gestures.pop();
+  }
+
+  let touchStateObjects = await asyncmap(gestures, async (gesture) => {
```

Comment:
Does this pass linting? JSHint has problems, as far as I know, with async functions using the fat arrow.